src/sentry/constants.py

```diff
@@ -216,3 +216,19 @@ def get_all_languages():
 KNOWN_DSYM_TYPES = {
     'application/x-mach-binary': 'macho'
 }
+
+
+class ObjectStatus(object):
```

Comment:
I like this. Would be nice if we could replace all the individual ones with this. Would it be worth doing a migration later to try and convert these all to use the same values?